www/kubl05/SP1/index.html

```diff
@@ -4,15 +4,16 @@
 <head>
     <meta charset="utf-8" />
     <meta name="description" content="Osobní webové stránky sloužící jako oslava života a díla Lukáše Kubíka aka lukazka">
-    <meta name="keywords" content="Lukáš Kubík, osobní stránky, školní projekt, lukazko">
-    <meta name="web_author" content="Lukáš Kubík, @lukazko">
+    <meta name="keywords" content="Lukáš Kubík, osobní stránky, školní projekt, lukazko, 4IZ268, Písek">
+    <meta name="author" content="Lukáš Kubík, @lukazko">
     <meta name="icons_author" content="Freepik, http://www.freepik.com">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Lukáš Kubík | Osobní stránky</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="img/favicons/favicon.png" type="image/png">
-    <link rel="shortcut icon" href="img/favicons/favicon.png" type="image/png">
+    <!--[if IE]><link rel="shortcut icon" href="img/favicons/favicon.ico"><![endif]-->
     <link rel="stylesheet" type="text/css" media="screen" href="css/styles.css" />
+    <link rel="stylesheet" type="text/css" media="print" href="css/styles.css" />
```

Comment:
tady chces oddelit styly pro tisk? vsak je to uz v jednom souboru

Reply:
bez toho metatagu mi to prohlížeč ignoruje..

Reply:
Myslim ze to je tim media=screen nad nim